tests/cephfs/cephfs_journal_tool/cephfs_journal_tool_journal_mode.py

```diff
@@ -146,6 +161,14 @@ def run(ceph_cluster, **kw):
         log.error(traceback.format_exc())
         return 1
     finally:
+        # start mds
+        for mds in mds_list:
+            fs_util.deamon_op(mds[0], "mds", "reset-failed", service_name=mds[1])
+            fs_util.deamon_op(mds[0], "mds", "start", service_name=mds[1])
```

Comment:
@julpark-rh  -- Can you also include "def check_deamon_status" to validate the expected daemon status after starting ??

Reply:
added